### PR TITLE
Add admin view for orders

### PR DESCRIPTION
**Pull request by seallard**
Not sure how to setup the visualization for cases in the order since it is not a model. Leaving it for later.

### CODE REVIEW

File: cg/server/admin.py [hunk 1]
```diff
@@ -448,6 +448,14 @@ class OrganismView(BaseView):
     column_searchable_list = ["internal_id", "name", "reference_genome"]
 
 
+class OrderView(BaseView):
+    """Admin view for Model.Order"""
+
+    column_default_sort = ("order_date", True)
+    create_modal = True
+    edit_modal = True
```
Comment (islean) [inline cg/server/admin.py:456]: Could we add that we can toggle "is_delivered" without editing the whole order?

**Comment by islean** [comment]
FYI, you can view all connected cases by editing an order:

image

File: cg/server/admin.py (hunk 1 above)
Comment (seallard) [reply cg/server/admin.py:456]: Not sure how to do that atm, will add later

**Review by islean — APPROVED** [review]
Looks good as a first rendition!

**Pull request merged**